xla/xla.proto

```diff
@@ -804,6 +804,9 @@ message DebugOptions {
   // source and target are located within a node(nvlink).
   bool xla_gpu_use_memcpy_local_p2p = 287;
 
+  // Use lld as a library for the linking step
+  bool xla_gpu_use_inprocess_lld = 389;
```

Comment:
XLA:GPU option names are sorted alphabetically. This line is out of order. Could you please put it before `xla_gpu_use_memcpy_local_p2p`?